sail/Search/Database.php

```diff
@@ -39,25 +39,25 @@ public function store(object|array $document, string $dataIndex = '')
      *
      * Delete a document by id
      *
-     * @param  string  $id
-     * @param  string  $dataIndex
+     * @param string $id
+     * @param string $dataIndex
      * @return void
      * @throws DatabaseException
      *
      */
     public function remove(string $id, string $dataIndex = '')
     {
         $search = new Search();
-        $search->remove($id);
+        $search->delete($id);
```

Comment:
isnt this changed for `remove` in the activerecord trait?

Reply:
I've tested and remove didn't work... 

It's because in the Search model we have that, 

`public function delete(string $id): void
    {
        $this->deleteOne(['document_id' => $id]);
    }
`
We don't delete by id, but by document_id.

Reply:
ahh true. Nevermind this comment then